rest/taskrouter/workflows/list/post/example-1/example-1.6.x.py

```diff
@@ -0,0 +1,47 @@
+# Download the Python helper library from twilio.com/docs/python/install
+from twilio.rest import Client
+import json
+
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+auth_token = "your_auth_token"
+workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+client = Client(account_sid, auth_token)
+
+sales_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+marketing_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+support_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+everyone_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+config = {
```

Comment:
@mosampaio It looks like the next-gen library still doesn't support JSON config generation. Please create an issue explaining this. These are [the missing classes](https://github.com/twilio/twilio-python/tree/master/twilio/task_router) that you can find in the legacy version. Please keep track of this snippet so we can modify it when the library supports it.